README.md

```diff
@@ -121,6 +121,16 @@ With this enabled you can use `<prefix> C-l` to clear the screen.
 
 Thanks to [Brian Hogan][] for the tip on how to re-map the clear screen binding.
 
+#### Intellectual Clear Screen
+
+Additionally, it is possible to make `<Ctrl-l>` to work as usual if current pane is full-width (this will also work well if that pane is Vim):
+
+``` tmux
+single_pane='test $(tmux list-panes | grep "(active)" | grep -o "\[[0-9]*x" | tr -d "[x") -eq $(tmux list-windows | grep "(active)" | grep -o " \[[0-9]*x" | tr -d "[x")'
+...
+bind -n C-l if-shell "$single_pane || $is_vim" "send-keys C-l" "select-pane -R"
```

Comment:
Nice, but it can be made easier:

`bind -n C-l if '[ #{pane_width} = #{window_width} ]' "send-keys C-l" "select-pane -R"`

(This also includes `window_zoomed_flag==1`.)